src/commands/syncTasks.ts

```diff
@@ -117,8 +118,10 @@ export class SyncTasksCommand extends LockableCommand {
         }
 
         const id = task.id ? ` [#${task.id}]` : '';
+        const assignee = task.assignee ? ` @${task.assignee}` : '';
+        const tags = task.tags ? ` #${task.tags.join(' #')}` : '';

```

Comment:
When `task.tags` is `[]` it's truthy while in this case we want to show the tags only if there is any so this could be replaced with `task.tags?.length` so that it's only true when there is at least one tag.